observablehq.config.js

```diff
@@ -3,6 +3,7 @@ import { getDateXDaysAgo } from './src/utils/date-utils.js'
 
 const startProviders = '2024-04-07'
 const startClients = '2025-02-25'
+const startAllocators = '2025-03-26'
```

Comment:
We're showing allocators for the last 31 days, should we change `startAllocators` in that case?

```suggestion
const startAllocators =  getDateXDaysAgo(31)
```

Reply:
I don't think this value is showing the last 31 days. just like `startClients` and `startProviders`

Reply:
I was referring to the file src/data/spark-allocators-rsr.json.js, where we retrieve data for the past 31 days and display it in a table. I was wondering whether it's necessary to fetch data for allocators that aren't currently being rendered. However, it turns out that we do need to fetch them—consider a case where someone has a direct link to an allocator that isn't displayed in the table.